### PR TITLE
modules: lvgl: simplify Kconfig memory pool size

### PR DESCRIPTION
**Pull request by faxe1008**
Took note of this while working on the demos, the memory pool size configs are needlesly confusing. There is this notion of blocks. But the chunking is not actually controllable for the users of sys_heap so there is not much point in complicating the Kconfig symbols.

### CODE REVIEW

File: modules/lvgl/Kconfig.memory [hunk 1]
```diff
@@ -31,28 +31,13 @@ choice LV_Z_MEMORY_POOL
...
-if LV_Z_MEM_POOL_SYS_HEAP
-
-config LV_Z_MEM_POOL_MIN_SIZE
-	int "Minimum memory pool block size"
-	default 16
-	help
-	  Size of the smallest block in the memory pool in bytes
-
-config LV_Z_MEM_POOL_MAX_SIZE
+config LV_Z_MEM_POOL_SIZE
```
Comment (jfischer-no) [inline modules/lvgl/Kconfig.memory:34]: MIN, MAX and BLOCKS are the traces from the past, https://github.com/zephyrproject-rtos/zephyr/pull/28611/files#diff-17227f3b6ec734be2bf8fdfc83c409931dc8f0e1586d528d6335ae237ad72031. You will probably also want to update prompt and help texts of this Kconfig option.

File: samples/modules/lvgl/demos/prj.conf [hunk 2]
```diff
@@ -3,7 +3,7 @@ CONFIG_LOG=y
 CONFIG_SHELL=y
 
 CONFIG_LVGL=y
-CONFIG_LV_Z_MEM_POOL_NUMBER_BLOCKS=8
+CONFIG_LV_Z_MEM_POOL_SIZE=16384
```
Comment (jfischer-no) [inline samples/modules/lvgl/demos/prj.conf:6]: Please squash in one commit, otherwise not bisectable.

**Review by jfischer-no — CHANGES_REQUESTED** [review]

File: modules/lvgl/Kconfig.memory (hunk 1 above)
Comment (faxe1008) [reply modules/lvgl/Kconfig.memory:34]: Oh yeah, right missed those.

File: samples/modules/lvgl/demos/prj.conf (hunk 2 above)
Comment (faxe1008) [reply samples/modules/lvgl/demos/prj.conf:6]: Sure thing.

**Review by gmarull — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

**Review by pdgendt — APPROVED** [review]

**Pull request merged**